Doc/tools/touch-clean-files.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+"""
+Touch files that must pass Sphinx nit-picky mode
+so they are rebuilt and we can catch regressions.
+"""
+
+from pathlib import Path
+
+# Input file has blank line between entries to reduce merge conflicts
+with Path("Doc/tools/clean-files.txt").open() as clean_files:
+    CLEAN = [
+        filename.strip()
```

Comment:
```suggestion
        Path(filename.strip())
```

May as well just do the Path conversion here, eh?